### DISCUSSION TITLE
Welcome to LeagueAkari Discussions!

### DISCUSSION THREAD

**Discussion by Hanxven** [LeagueAkari/LeagueAkari#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jayesslee1** [comment]
问客户端里的玩家标记功能有什么用？对方改名之后还有用吗？被标记的玩家下次再排到会不会有提示？这个标记是仅自己可见 还是所有用户都能看到

**Reply by Hanxven** [reply]
您好！客服 Rabi 为您服务。

## 标记功能的用途？

字面意义的标记，可以在**本地**记录你对 ta 的看法或意见。

标记被存储在本地数据库，位于 `C:\Users\{username}\AppData\Roaming\league-akari\LeagueAkari.db`。

## 对方改名后还有用吗？

标记基于全球唯一的 PUUID，和玩家名称无关。

## 再次排到会怎样？

如果作为队友，会在英雄选择阶段的聊天框中发送一条**仅自己可见**的信息，包含当时标记的内容。

同时，在对局面板上，被标记玩家会有一个额外的标签 “被标记”，悬停或进入其战绩页面即可查看标记内容。

## 标记的可见性？

目前，League Akari 的所有数据都位于[**本地**](https://github.com/Hanxven/LeagueAkari/blob/main/src/main/shards/storage/entities/SavedPlayers.ts)，没有共享机制。

并不存在一个远程服务器来存储这些内容。

## 其他东西

在下个版本中，会加入对标记的导出和导入功能。

由于设计理念及其他多方面的考虑，本工具在云端存储的实现上采取了相对保守的策略。

**Reply by Jayesslee1** [reply]
谢谢！答疑非常专业！

**Comment by TeresaLi007** [comment]
The latest updated version 1.3.5, when opened with the Lenovo Legion y9000p (win11), will automatically show full screen, and the display part is only a small part in the middle, which makes it impossible to zoom in normally for use. So now I have to use the old version. Are there any friends who have the same situation as me?Hope to repair it as soon as possible.

**Reply by Hanxven** [reply]
Issue fixed since v1.4.0.

**Comment by HomNambest** [comment]
突然发现软件可以自动更新，这个功能太赞了

**Comment by LxxCute2** [comment]
上次版本更新之后就无法看到正在进行对局的成员信息了，以前是可以看到好友游戏对局内十个人的id以及开黑信息的，这个功能还蛮好用的，为什么现在删除了呢T_T，当时位置大概就是在这
<img width="353" height="411" alt="51e3d907-67fb-44db-a333-b3c1e479239d" src="https://github.com/user-attachments/assets/affea6d6-b247-486b-8330-81c4e7b7f86f" />

**Reply by Hanxven** [reply]
相关接口已被官方禁用。

**Reply by LxxCute2** [reply]
好吧，谢谢你的回复！

**Comment by HomNambest** [comment]
下个版本的更新大约在多久呢，有点期待

**Comment by Chester0317** [comment]
Does it work for the EUW server?
